editor.html

```diff
@@ -281,6 +281,7 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
                         <div class="buttons_menu_item"><a id="docs-link" class="action" title="View the documentation for MicroPython" href="https://microbit-micropython.readthedocs.io/" target="_blank">Documentation</a></div>
                         <div class="buttons_menu_item"><a id="help-link" class="action" title="Open help for this editor in a new tab" href="help.html" target="_blank" >Help</a></div>
                         <div class="buttons_menu_item"><a id="support-link" class="action" title="Get support for your micro:bit in a new tab" href="https://support.microbit.org/support/home" target="_blank">Support</a></div>
+			<div class="buttons_menu_item"><a id="feedback-link" class="action" title="Provide some feedback on the editor" href="https://support.microbit.org/en/support/tickets/new" target="_blank">Feedback</a></div>
```

Comment:
Can we hide this entry and only show it in the beta editor like we do with the Issue Tracker?

Reply:
@microbit-mark I'll be editing how we show and hide experimental features in a new branch, so I'll merge this and do that at the same time.